sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+PYTHON_REQ_USE="sqlite(+)"
+DISTUTILS_SINGLE_IMPL=1
+
+inherit desktop distutils-r1
+
+DESCRIPTION="Clean junk to free disk space and to maintain privacy"
+HOMEPAGE="https://www.bleachbit.org"
+COMMIT="f0f93b4427a3607898da18ad888e32a30aed654e"
+SRC_URI="https://github.com/bleachbit/bleachbit/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	$(python_gen_cond_dep '
+	dev-python/chardet[${PYTHON_MULTI_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_MULTI_USEDEP}]
+	dev-python/scandir[${PYTHON_MULTI_USEDEP}]') "
```

Comment:
Minor nit. Please make this consistent with BDEPEND, i.e. move quotation mark to next line.